crates/bevy_window/src/window.rs

```diff
@@ -266,38 +277,43 @@ impl Window {
         self.resolution.scale_factor()
     }
 
-    /// The cursor position in this window
+    /// The cursor position in this window in logical pixels.
+    ///
+    /// Returns `None` if the cursor is outside the window area.
     #[inline]
     pub fn cursor_position(&self) -> Option<Vec2> {
         self.internal
             .physical_cursor_position
             .map(|position| (position / self.scale_factor()).as_vec2())
     }
 
-    /// The physical cursor position in this window
+    /// The cursor position in this window in physical pixels.
+    ///
+    /// Returns `None` if the cursor is outside the window area.
```

Comment:
I appreciate it's a bit out of scope here, but `RelativeCursorPosition` uses negative values for this, this is a bit inconsistent. Is there really a value to losing the cursor position information relative to the window when it's not hovering, when the "is hovering" information is already readily available from the same `Window` component (possibly needs helper for one-liner`, but I think is easily calculated with the window size and the relative cursor position, provided it's never `None`.

Also, when implementing some form of drag and drop, you want to continue dragging even if the cursor is outside the window in one direction but not the other. Here you can't.

Reply:
(I resolved this conversation because I don't think it has to do with the documentation, if you want to copy/link your comment in https://github.com/bevyengine/bevy/issues/8865 feel free to do so ^^)